docs/apm/service-maps.asciidoc

```diff
@@ -1,37 +1,50 @@
+[role="xpack"]
 [[service-maps]]
 === Service maps
 
 beta::[]
 
-A service map is a real-time diagram of the interactions occurring in your application’s architecture.
-It allows you to easily visualize data flow and high-level statistics, like average transaction duration,
-requests per minute, errors per minute, and metrics, allowing you to quickly assess the status of your services.
+A service map is a real-time visual representation of the instrumented services in your application’s architecture.
+It shows you how these services are connected, along high-level metrics like average transaction duration,
+requests per minute, and errors per minute, that allow you to quickly assess the status of your services.
 
-Our beta offering creates two types of service maps:
+We currently surface two types of service maps:
 
-* Global: All services and connections are shown.
-* Service-specific: Selecting a specific service will highlight it's connections.
+* Global: All services instrumented with APM agents and the connections between them are shown.
+* Service-specific: Highlight collections for a selected service.
 
 [role="screenshot"]
 image::apm/images/service-maps.png[Example view of service maps in the APM app in Kibana]
 
+[float]
+[[service-maps-how]]
+=== How do service maps work?
+
+Service maps rely on distributed traces to draw connections between services.
+As {apm-overview-ref}/distribute-tracing.html[distributed tracing] is enabled out-of-the-box for supported technologies, so are service maps.
+However, if a service isn't instrumented,
+or a `traceparent` header isn't being propagated to it,
```

Comment:
"or is unable to propagate a `traceparent` header" Because you can see external service and database calls and won't see things like load balancers because they themselves don't explicitly propagate the traceparent header. Maybe we can specify what things can't propagate headers explicitly - e.g. load balancers, databases, message queues.